### ISSUE TITLE
[BUG] Dashboard saved field issues after upgrade (LME 0.3 --> 0.4)

### ISSUE THREAD

**Issue by joncojonathan** [ukncsc/lme#127]
Prior to upgrading to v0.4 ([following these instructions](https://github.com/ukncsc/lme/blob/master/docs/upgrading.md)) my dashboards showed data.  For example, on the _User investigator_ dashboard I could view the _Filter by username_ widget.  After upgrading, these widgets now display a red _Error_ icon.  Hovering on the icon gives me advice such as:

```
[esaggs] > Saved field "user.name" of index pattern "winlogbeat-*" is invalid for use with the "Terms" aggregation.  Please select a new field.
```

or 

```
[esaggs] > Saved field "process.executable" of index pattern "winlogbeat-*" is invalid for use with the "Terms" aggregation.  Please select a new field.
```

I have re-indexed my data (and deleted the old indices) per the instructions.   The latest rules were also imported by the script and the web UI per the instructions.I also imported the latest dashboards via the script (doing so dumped a load of JSON to the terminal, and it was unclear if that was an indicator of success or failure). 

Concerned the dashboard import didn't complete successfully I did installed (and overwrote) the dashboards via the UI per [4.4 Troubleshooting - Manual Dashboard Install](https://github.com/ukncsc/lme/blob/master/docs/chapter4.md#44-troubleshooting---manual-dashboard-install).  This doesn't seem to have made a difference.

**To Reproduce**
1. Upgrade to LME 0.4 from 0.3
2. Go to menu > Dashboard
3. Choose _Security dashboard_
4. Choose _User Investigator_
5. Note the _error_ icons on _Filter by username_ and other widgets

**Expected behavior**
Dashboards and widgets should show correct data, without errors.

**Linux Server (please complete the following information):**
 - Docker: 20.10.12, build e91ed57
 - Docker compose stack file version: [e.g. version 0.1]
 - Linux: Ubuntu 20.04.3 LTS  (issue was also present on Ubuntu 18.04 LTS)
 - Logstash Version: 0.4 - 24/03/21

**Additional context**
See also #128 which also occurred after the same upgrade, and may be related.

**Comment by adam-ncc** [comment]
Likely the same issue as #128.

**Comment by joncojonathan** [comment]
I can confirm the fix for this was the same as #128.  The specific steps I took can be found here: https://github.com/ukncsc/lme/issues/128#issuecomment-1074018893.
